python/vowpalwabbit/pyvw.py

```diff
@@ -1513,6 +1513,9 @@ def check_col_type(self, df):
         expected_type = list(self.expected_type)
         if str in expected_type:
             expected_type.append(object)
+        if int in expected_type:
+            import numpy as np
```

Comment:
Why not put this include at the top of the file?

Reply:
Whoops, I added that there to test because I was too lazy to scroll back up to the top. Let me move that haha